README.md

```diff
@@ -0,0 +1,35 @@
+# opentracing-compat
+
+A compatibility layer bridging [`defn-traced`][github-clj-newrelic] with the
+OpenTracing API.
+
+## Quick start
+
+Use it just like [`defn-traced`][github-clj-newrelic]:
+
+```clojure
+(ns user
+  (:require [democracyworks.opentracing.compat :refer [defn-traced]]))
+
+(defn-traced sleep-for
```

Comment:
I get that this is a temporary drop in to replace the _other_ `defn-traced`, but I do hope long term we're considering something less messy for tracing in the future.

Reply:
You shouldn't use this for anything new, just use OpenTracing directly. Or is this a joke about `sleep-for`...?

Reply:
Nah, more just on `defn-traced`, and hoping that we'll eventually get away from it.